synapse/lib/view.py

```diff
@@ -275,6 +275,27 @@ def init2(self):
     def isafork(self):
         return self.parent is not None
 
+    def isForkOf(self, viewiden):
+        view = self.parent
+        while view is not None:
+            if view.iden == viewiden:
+                return True
+            view = view.parent
+        return False
+
+    async def calcForkLayers(self):
```

Comment:
If this is only safe from the nexus handler can we mark it as a private method?

Reply:
Do you just mean with an `_` ?

Reply:
Yup